lldb/source/Target/Process.cpp

```diff
@@ -6410,12 +6410,20 @@ GetCoreFileSaveRangesStackOnly(Process &process,
     if (!reg_ctx_sp)
       continue;
     const addr_t sp = reg_ctx_sp->GetSP();
+    const size_t red_zone = process.GetABI()->GetRedZoneSize();
```

Comment:
Asking for my understanding:
This will only really do anything if the size of the red zone is greater than 0 right? Otherwise, this change is a no-op. Is that right?

Reply:
Correct, but on systems where it would be greater than 0, we ensure we capture it for the memory dump as well.